### PR TITLE
Add Lighthouse checks

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Trying to prevent future performance regressions

### CODE REVIEW

File: .github/workflows/lighthouse.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: CI
+on: [push]
+jobs:
+  lighthouseci:
+    runs-on: ubuntu-latest
```
Comment (roj1512) [inline .github/workflows/lighthouse.yml:5]:
```suggestion
name: CI

on:
  push:

jobs:
  lighthouseci:
    runs-on: ubuntu-latest
```

File: .github/workflows/lighthouse.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: npm install && npm install -g @lhci/cli@0.11.x
+        working-directory: site/
+
+      - name: Build site
```
Comment (roj1512) [inline .github/workflows/lighthouse.yml:18]:
```suggestion
      - name: Build
```

**Review by roj1512 — APPROVED** [review]

File: .github/workflows/lighthouse.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+name: Lighthouse
+
+on:
+  push:
+
+jobs:
+  lighthouseci:
```
Comment (roj1512) [inline .github/workflows/lighthouse.yml:7]: Forgot this.

```suggestion
  lighthouse:
```

**Comment by KnorpelSenf** [comment]
It would be nice if this actually worked

**Comment by KnorpelSenf** [comment]
I have no idea how to fix this

**Comment by KnorpelSenf** [comment]
If you do, feel free to go ahead. Perf matters.

**Comment by KnorpelSenf** [comment]
If you don't, we can also just close this. Since we fixed the acute problems in #799, there's no immediate need for this anymore.

**Comment by KnorpelSenf** [comment]
I'll leave it up to you and stop spamming

**Comment by KnorpelSenf** [comment]
:)

**Comment by roj1512** [comment]
(:

**Comment by roj1512** [comment]
I will see.

**Comment by KnorpelSenf** [comment]
Okay

**Comment by KnorpelSenf** [comment]
Cool

**Comment by KnorpelSenf** [comment]
I'm kinda intrigued if this works

**Comment by KnorpelSenf** [comment]
And yet I can't be bothered to fix this myself (sorry)

**Comment by KnorpelSenf** [comment]
Did I say I'll stop spamming?

**Comment by KnorpelSenf** [comment]
I take that back.

**Pull request closed without merging**